src/main/java/com/clouway/chita/ChitaHttpClient.java

```diff
@@ -95,7 +95,11 @@ public HttpResponse execute(HttpRequest request) {
 
         }
 
-        inputStream = conn.getInputStream();
+        try {
+          inputStream = conn.getInputStream();
```

Comment:
Problem from the issue was coming from ` conn.getInputStream() ` which was throwing an IOException when the response from the server contained only an error code.
The exception was being caught [on this line](https://github.com/clouway/chita/pull/13/files#diff-dbe726534ebbf5921a4b3d05830f4671R110) and printed out.
In turn a ` new HttpResponse(-1, ""); ` was being returned([from this line](https://github.com/clouway/chita/pull/13/files#diff-dbe726534ebbf5921a4b3d05830f4671R125))